src/main/java/net/openhft/chronicle/wire/Wires.java

```diff
@@ -674,37 +676,69 @@ public static WireOut writeDate(Date date, ValueOut out) {
         }
 
         public static Date parseDate(ValueIn in) {
-            String text = in.text().trim();
-            try {
-                return new Date(Long.parseLong(text));
-            } catch (NumberFormatException nfe) {
-                try {
-                    // Make sure to always synchronize on these fields in exactly this order or else there might be deadlocks
-                    synchronized (SDF_2) {
-                        return SDF_2.parse(text);
+            final String text = in.text().trim();
+            if (text.length() < 1) {
+                throw new IORuntimeException("At least one character (e.g. '0') must be present in order to deserialize a Date object");
+            }
+            final char firstChar = text.charAt(0);
+
+            // Check if it is a number
+            if (firstChar == '+' || firstChar == '-' || Character.isDigit(firstChar)) {
+                boolean isAllNum = true;
+                for (int i = 1; i < text.length(); i++) {
+                    if (!Character.isDigit(text.charAt(i))) {
```

Comment:
This may potentially be slow with long inputs, slower than fail-fast of other formats, so I'm not sure it should not go last.

Reply:
I think that other valid formats have spaces or `-` relatively early so this should complete quickly for valid inputs.